client/signup/steps/site-or-domain/index.jsx

```diff
@@ -50,27 +51,33 @@ export default class SiteOrDomain extends Component {
 	}
 
 	getChoices() {
+		const { translate } = this.props;
+
 		return [
 			{
 				type: 'page',
-				label: 'New site',
+				label: translate( 'New site' ),
 				image: <NewSiteImage />,
-				description: 'Choose a theme, customize, and launch your site. Free domain included with all plans.'
+				description: translate( 'Choose a theme, customize, and launch your site. Free domain included with all plans.' )
 			},
 			{
 				type: 'domain',
-				label: 'Just buy a domain',
+				label: translate( 'Just buy a domain' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Buy Domain' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).